src/content/docs/ko/recipes/i18n.mdx

```diff
@@ -0,0 +1,512 @@
+---
+title: 국제화 기능 추가
+description: 동적 라우팅 및 콘텐츠 컬렉션을 사용하여 Astro 사이트에 국제화 지원을 추가하세요.
+type: recipe
+i18nReady: true
+---
+
+import FileTree from '~/components/FileTree.astro'
+import ReadMore from '~/components/ReadMore.astro'
+import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
+
+이 레시피에서는 콘텐츠 컬렉션과 동적 라우팅을 사용하여 자체 국제화 (i18n) 솔루션을 구축하고 콘텐츠를 다양한 언어로 제공하는 방법을 배웁니다.
+
+:::tip
+v4.0에서 Astro는 기본 및 지원되는 언어를 구성할 수 있게 해주는 i18n 라우팅에 대한 내장 지원을 추가했으며, 국제 청중에게 서비스를 제공하는 데 도움이 되는 귀중한 도우미 기능을 포함하고 있습니다. 대신 이 기능을 사용하려면 [국제화 가이드](/ko/guides/internationalization/)를 참조하여 이러한 기능에 대해 알아보세요.
+:::
+
+이 예시는 자체 하위 경로에서 각 언어를 제공합니다. 영어의 경우 `example.com/en/blog`, 프랑스어의 경우 `example.com/fr/blog`입니다.
+
+다른 언어와 달리 기본 언어가 URL에 표시되지 않도록 하려면 아래의 [기본 언어를 숨기는 방법](#url에서-기본-언어-숨기기)을 참조하세요.
```

Comment:
```suggestion
다른 언어와 달리 기본 언어가 URL에 표시되지 않도록 하려면 아래의 [기본 언어를 숨기는 방법](/ko/recipes/i18n/#url에서-기본-언어-숨기기)을 참조하세요.
```

Reply:
Both links seem to go to the same page, is there any benefit to adding the `/ko/recipes/i18n/` path before the existing link?

Reply:
Not really, I don't think that there's a benefit. But just to be compliant with the `en/` locale just in case in the near future it is fixed, you won't be able to remove that update from the tracker cause you've already fixed it right now. You'll need to add a space somewhere so it can count as update so the tracker can remove it out of your pending updates.